wrk_bench.rb

```diff
@@ -20,7 +20,7 @@
   wrk_binary: "wrk",
   wrk_connections: 100,
   url: "http://127.0.0.1:PORT/simple_bench/static",
-  server_pre_cmd: "bundle install && bundle exec rake db:migrate",
```

Comment:
Is this removed because the before_worker_cmd also does it (https://github.com/noahgibbs/rsb/pull/2/files#diff-617fd952ee839f5b80844420043503e7L121)? If so, keep in mind that the top-level Gemfile for RSB doesn't necessarily have all the gems that the Rails test app does.

Reply:
Cool. This seems to be working fine in practice.

Reply:
Yes, I think `before_worker_cmd` is always called before `server_pre_cmd`.
Is `wrk_bench.rb` still maintained? I tried to keep it correct but didn't test with it.

Reply:
wrk_bench.rb is outdated and intended to disappear soon (there's a comment about that up-top), so no worry about that.